modules/MuKeyboard.js

```diff
@@ -0,0 +1,252 @@
+/* jshint laxbreak: true */
+
+new Object({
+  moduleMeta: { /* Module infos */
+    displayName: 'MuKeyboard',
+    version: 1.0,
+    author: 'MrMoDDoM'
+  },
+
+  functionMap: { /* Functions */
+    REM: function(argList, µ) {
+      argList.shift();
+
+      µ.setData('nodelay', true);
+      return '  // ' + argList.join(' ') + '\n';
+    },
+
+    TYPE: function(argList, µ) {
+      argList.shift();
+
+      if(argList.length < 1) {
+        µ.throwError("No argument provided for TYPE");
+        return;
+      }
+
+      if(µ.keyMap[argList[0]] !== undefined) {
+        return '  typeKey(\'' + µ.keyMap[argList[0]] + '\');\n\n';
+      } else {
+        µ.throwError("Unknown command or key: '" + argList[0] + "'");
+        return;
+      }
+    },
+
+    STRING: function(argList, µ) {
```

Comment:
Please remove the  µ(mu) character from this. It's an anti-pattern to use non-alphanumeric variable names. Additionally if you use it in a string, please use the proper escape code for it.

Reply:
Just realized this is in the core codebase, sorry. CC @Nurrl